book/tables/pharmacokinetic/pkct01.qmd

```diff
@@ -26,78 +26,116 @@ adpc_1 <- adpc %>%
     AVALCAT1 = as.factor(AVALCAT1)
   ) %>%
   filter(ACTARM == "A: Drug X", PARAM == "Plasma Drug X") %>%
-  mutate(ACTARM = factor(ACTARM, levels = c("A: Drug X"))) %>%
-  select(NFRLT, ACTARM, VISIT, AVAL, PARAM, AVALCAT1)
+  select(NFRLT, ACTARM, VISIT, AVAL, PARAM, AVALCAT1) %>%
+  var_relabel(NFRLT = "Nominal Time from First Dose (hr)")
```

Comment:
is this a typo in rcd?

Reply:
No, just in the template - a correct (slightly different) label is used in rcd.